Update developing-with-ce.asciidoc (#197)
vaadin/docs: articles/ce/developing-with-ce.asciidoc

---
title: Licensing and Pricing Model
order: 3
layout: page
---

[[ce.developing]]

> = Licensing

Reviewer: ```suggestion
= Licensing and Pricing Model
```
I **think** that the first header should always be the same as the title.

Reviewer: @jouni was there some rule like this?

Collaboration Engine is licensed under the https://vaadin.com/license/cvrl-1.0[Commercial Vaadin Runtime License (CVRL)].
You can try Collaboration Engine out and develop applications with it without charge.
When you take your application to production, you need a valid license file.
To get started, follow the <<tutorial#,Quick Start Guide>> to learn about adding Collaboration Engine to your Vaadin application, and using its high-level APIs.

While your application is in development mode, you can use all the features of Collaboration Engine without restrictions.
You do not need a development license or a separate trial.
Development mode is the default mode of a Vaadin application and is used when not explicitly configured to do a production build.

[[ce.developing.pricing-model]]

> == Billing model

Reviewer: ```suggestion
== Billing Model
```
Always Title Case Headers

Collaboration Engine is billed at run-time, based on the number of unique end users per month using collaboration features in an application.

There are two options for taking an application that uses Collaboration Engine into production:

* Free "Universal" License (works with any Vaadin application)
* Commercial License (requires an active commercial Vaadin subscription)

[[ce.developing.free-universal-license]]
== Free "Universal" License

> The "Universal" License is offered free to all Vaadin developers. It allows 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month. Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.

Reviewer: ```suggestion
The "Universal" License is offered free to all Vaadin developers.
It allows 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month.
Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.
```
One sentence per line.

> The "Universal" License is offered free to all Vaadin developers. It allows 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month. Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.

Reviewer: > Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.

Some words are missing here. (and text checker did not like the word obtain)

Author: The "Universal" License is free for all Vaadin developers. It allows up to 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month. Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to use this license.

Author: @Peppe Would this work? Still don't know how to add suggestions... Sorry.

Reviewer: Sounds good. Will edit suggestion.

Reviewer: Added a new edit suggestion.

You can try Collaboration Engine out and develop applications with it without charge.
When you take your application to production, you need a valid license file.
To get started, follow the <<tutorial#,Quick Start Guide>> to learn about adding Collaboration Engine to your Vaadin application, and using its high-level APIs.

While your application is in development mode, you can use all the features of Collaboration Engine without restrictions.
You do not need a development license or a separate trial.
Development mode is the default mode of a Vaadin application and is used when not explicitly configured to do a production build.

[[ce.developing.pricing-model]]
== Billing Model

Collaboration Engine is billed at run-time, based on the number of unique end users per month using collaboration features in an application.

There are two options for taking an application that uses Collaboration Engine into production:

* Free "Universal" License (works with any Vaadin application)
* Commercial License (requires an active commercial Vaadin subscription)

[[ce.developing.free-universal-license]]
== Free "Universal" License

> The "Universal" License is offered free to all Vaadin developers.
> It allows 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month.
> Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.

Reviewer: ```suggestion
The "Universal" License is free for all Vaadin developers. It allows up to 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month.
Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to use this license.
```
